usecases/traverser/traverser_aggregate.go

```diff
@@ -35,6 +35,23 @@ func (t *Traverser) Aggregate(ctx context.Context, principal *models.Principal,
 
 	inspector := newTypeInspector(t.schemaGetter)
 
+	if params.NearVector != nil || params.NearObject != nil || len(params.ModuleParams) > 0 {
+		className := params.ClassName.String()
+		err = t.nearParamsVector.validateNearParams(params.NearVector,
+			params.NearObject, params.ModuleParams, className)
+		if err != nil {
+			return nil, err
+		}
+		searchVector, err := t.nearParamsVector.vectorFromParams(ctx,
+			params.NearVector, params.NearObject, params.ModuleParams, className)
+		if err != nil {
+			return nil, err
+		}
+		params.SearchVector = searchVector
+		params.Certainty = t.nearParamsVector.extractCertaintyFromParams(params.NearVector,
```

Comment:
I think also that we should throw an error if no `certainty` is passed in `Aggregate` query in `near<>` param, bc in `Aggregate` only `near<>.certainty` cut off makes the results differ.

Reply:
please ignore this comment as I saw that this check is being made in `filtered.go` `searchByVector` method